Modules/Devices/src/Devices.Infrastructure/PushNotifications/FirebaseCloudMessaging/FirebaseCloudMessagingConnector.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Text.Json;
+using Backbone.Modules.Devices.Application.Extensions;
+using Backbone.Modules.Devices.Application.Infrastructure.PushNotifications;
+using Backbone.Modules.Devices.Infrastructure.PushNotifications.AzureNotificationHub;
+using Backbone.Modules.Devices.Infrastructure.PushNotifications.DirectPush;
+using FirebaseAdmin.Messaging;
+
+namespace Backbone.Modules.Devices.Infrastructure.PushNotifications.FirebaseCloudMessaging;
+public class FirebaseCloudMessagingConnector : IPnsConnector
+{
+    public FirebaseCloudMessagingConnector()
+    { }
+
+    public async Task Send(IEnumerable<PnsRegistration> registrations, object notification)
+    {
+        var recipients = registrations.Select(r => r.Handle.Value).ToList();
+        var recipientsLists = recipients.Split(500);
+        var data = new Dictionary<string, string>();
```

Comment:
Why do you just send an empty object as `data` for every notification? Am I missing something?